cmd/troubleshoot/cli/run_nocrd.go

```diff
@@ -74,12 +74,12 @@ func runTroubleshootNoCRD(v *viper.Viper, arg string) error {
 				switch msg := msg.(type) {
 				case error:
 					c := color.New(color.FgHiRed)
-					c.Println(fmt.Sprintf("%s * %v", cursor.ClearEntireLine(), msg))
+					c.Println(fmt.Sprintf("%s\r * %v", cursor.ClearEntireLine(), msg))
```

Comment:
Changes in this file just to make output a little prettier... Some lines were being merged together.